src/main/java/lv/acodemy/homework/SecondHomeWork.java

```diff
@@ -0,0 +1,73 @@
+package lv.acodemy.homework;
+
+import java.util.Arrays;
+public class SecondHomeWork {
+    public static void main(String[] args) {
+
+        //1 Task:
+        //1. Create String array that will contain 10 European countries and include "Latvia" in this array
+        //1.1 Iterate through array and print all values
+        //1.2 Iterate through array and add following logic: IF "Latvia" exist in the
+        //array print: "Latvia is my motherland"
+
+        String[] country = {"Estonia", "Lithuania", "Latvia",
+                "Poland", "Thailand", "France", "Denmark", "UK", "Norway", "Argentina"};
+        for (int i = 0; i < country.length; i++) {
+            if (country[i].equals("Latvia")) {
+                System.out.println("Latvia is my motherland");
+            } else {
+                System.out.println(country[i]);
+            }
+        }
+        System.out.println("----------");
+
+        //2 Task:
+        //2. You have following sentence: "Sometimes it pays to stay in bed
+        //on Monday, rather than spending the rest of the week debugging Monday’s code.”
+        //2.1 Put each word into a String array.
+        //2.1 Using Iteration (Loops) print all values from array so as the result will be printed sentence (one line)
+
+        String[] words = {"Sometimes","it", "pays", "to", "stay", "in", "bed",
+                "on", "Monday", "rather", "than", "spending", "the", "rest",
+                "of", "the", "week", "debugging", "Monday’s", "code"};
+        for (int i = 0; i < 1;i++) {
+            System.out.println(Arrays.toString(words));
```

Comment:
Not right solution. You have to iterate through each word from the array and print in in one line :)